api/src/certification/evaluation/routes.js

```diff
@@ -1,6 +1,7 @@
 import * as companionAlert from '../evaluation/application/companion-alert-route.js';
 import * as certificationCourseRoutes from './application/certification-course-route.js';
+import * as certificationRoutes from './application/certification-route.js';
 
-const certificationEvaluationRoutes = [companionAlert, certificationCourseRoutes];
+const certificationEvaluationRoutes = [companionAlert, certificationCourseRoutes, certificationRoutes];
```

Comment:
proposition de renommage
```suggestion
const certificationEvaluationRoutes = [companionAlert, certificationCourseRoutes, certificationAdminRoutes];
```